ca/django_ca/widgets.py

```diff
@@ -118,14 +119,19 @@ def decompress(
     ) -> typing.List[typing.Union[str, typing.List[str]]]:
         if value is None:  # pragma: no cover
             return ["", "", "", "", "", ""]
-
+        
+        # Multiple OUs are not supported in webinterface
```

Comment:
Minor copy/paste issue: you still refer to OUs here ;-)

Reply:
Whoops – fixed heh